### PR TITLE
Update EIP-1 to not use Issues for EIP number

### PR DESCRIPTION
**Pull request by poojaranjan**
Update EIP-1 to discourage use of Issues for "discussion-to" and "EIP number"

### CODE REVIEW

File: EIPS/eip-1.md [hunk 1]
```diff
@@ -229,7 +229,7 @@ If the EIP isn't ready, the editor will send it back to the author for revision,
 
 Once the EIP is ready for the repository, the EIP editor will:
 
-- Assign an EIP number (generally the PR number or, if preferred by the author, the Issue # if there was discussion in [the Issues section of this repository](https://github.com/ethereum/EIPs/issues) about this EIP)
+- Assign an EIP number (generally the PR number)
```
Comment (axic) [inline EIPS/eip-1.md:232]:
```suggestion
- Assign an EIP number (generally the PR number, but the decision is with the editors)
```

Comment (MicahZoltu) [reply EIPS/eip-1.md:232]: I would also like to see the change proposed here by Alex, but if you are against it @poojaranjan we can merge this as is (it is an improvement even without Alex's suggestion) and then debate the addition of Alex's change to a separate PR.

**Review by MicahZoltu — APPROVED** [review]

File: EIPS/eip-1.md (hunk 1 above)
Comment (poojaranjan) [reply EIPS/eip-1.md:232]: I am good!

**Review by lightclient — APPROVED** [review]
LGTM

**Pull request merged**